### PR TITLE
Updated README.md (added table of contents) and updated contributors.md

### PR DESCRIPTION
**Pull request by KugelPfusch**
please add label hacktoberfest-accepted and merge :)

### CODE REVIEW

File: contributors.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ Simply edit this file and add yourself to the list in the form of a pull request
 - Misty/EASTER EGG GOD
 - Daniel4-Scratch
 - Electro-Corp
+- Kugelpfusch
```
Comment (rgantzosonscratch) [inline contributors.md:10]: We don't use the `contributors.md` file anymore, and it will likely be deleted soon. Adding your name is unnecessary.

Comment (rgantzosonscratch) [inline contributors.md:10]: We don't use the `contributors.md` file anymore, and it will likely be deleted soon. Adding your name is unnecessary.

**Review by rgantzosonscratch — CHANGES_REQUESTED** [review]
Please remove your changes to the `contributors.md` file.

**Comment by KugelPfusch** [comment]
Done

**Comment by rgantzosonscratch** [comment]
> Done

Thanks. Also, I'm not sure what this is about, but your changes to the README file make the logo float left, rather than it being centered.

**Comment by KugelPfusch** [comment]
> > Done
> 
> Thanks. Also, I'm not sure what this is about, but your changes to the README file make the logo float left, rather than it being centered.

I didn't change anything there. Maybe its just the github preview? My Visual Studio Code extension shows it to me centered. I think its just the preview

**Comment by rgantzosonscratch** [comment]
> > > Done
> > 
> > 
> > Thanks. Also, I'm not sure what this is about, but your changes to the README file make the logo float left, rather than it being centered.
> 
> I didn't change anything there. Maybe its just the github preview? My Visual Studio Code extension shows it to me centered. I think its just the preview

Alright, thanks.

**Review by rgantzosonscratch — APPROVED** [review]

**Pull request merged**